files/default/neutron-ha-tool.py

```diff
@@ -23,6 +23,7 @@
 
 import argparse
```

Comment:
GitHub doesn't let me review the commit message directly, but it should say "its" not "it's" ;-)  "it's" always means "it is".